build.sh

```diff
@@ -99,15 +112,38 @@ UP=$(find $FILENAME -name "*.up.*.sh" -print)
 codesign -f -s "$SIGNINGIDENTITY" "$UP"
 
 echo ""
-read -p "Create disk image (requires DropDMG license) (y/n)?" choice
+
+INSTALLERFILENAME="eduVPN-Installer$(date +"%Y-%m-%d-%H:%M:%S.")dmg"
+
+
+create-dmg \
+--volname "eduVPN" \
+--volicon "icon.icns" \
+--background "background.png" \
+--window-pos 475 350 \
+--window-size 475 350 \
+--icon-size 100 \
+--icon "eduVPN.app" 100 155 \
+--hide-extension "eduVPN.app" \
+--app-drop-link 370 155 \
+ $INSTALLERFILENAME \
+$FILENAME"/eduVPN.app"
+codesign -f -s "$SIGNINGIDENTITY" $INSTALLERFILENAME
+
+echo ""
+echo "Done! You can now upload the files in the updates folders to your file server. Also remember to merge the release branch into master and tag it."
+
+
+
+read -p "Create another disk image (requires DropDMG license) (y/n)?" choice
 case "$choice" in
   y|Y ) ;;
   n|N ) exit 0;;
   * ) echo "Invalid response."; exit 0;;
 esac
 
 echo ""
-echo "Creating a disk image"
+echo "Creating a disk image using DropDMG"
```

Comment:
if createdmg works, dropdmg can probably be fully removed.

Reply:
HI François Boss,
Ok I am removing dropDMG completely